### ISSUE TITLE
Version wasn't bumped in lib/json/version.rb for 2.4.0

### ISSUE THREAD

**Issue by dentarg** [flori/json#456]
Should have been changed here?

https://github.com/flori/json/blob/c30183311113908e697d05f873264aad246f24c8/lib/json/version.rb#L4

```
$ cat Gemfile Gemfile.lock
# frozen_string_literal: true
source "https://rubygems.org"
gem "json", "2.4.0"
GEM
  remote: https://rubygems.org/
  specs:
    json (2.4.0)

PLATFORMS
  ruby

DEPENDENCIES
  json (= 2.4.0)

BUNDLED WITH
   2.1.4

$ bundle exec ruby -rjson -e 'p JSON::VERSION'
"2.3.1"
```

**Comment by mmcnickle-float** [comment]
And did a changelog entry get missed in the release too?

**Comment by hsbt** [comment]
Thanks, It's my bad. I fixed it and released json-2.4.1 now.
